RadixWallet/Core/SharedModels/P2P/Dapp/Models/InteractionResponse/RequestResponses/PersonaDataRequestResponseItem.swift

```diff
@@ -2,40 +2,19 @@
 // MARK: - P2P.Dapp.Response.WalletInteractionSuccessResponse.PersonaDataRequestResponseItem
 extension P2P.Dapp.Response.WalletInteractionSuccessResponse {
 	public struct PersonaDataRequestResponseItem: Sendable, Hashable, Encodable {
-		public let name: PersonaData.Name?
-		public let dateOfBirth: PersonaData.DateOfBirth?
-		public let companyName: PersonaData.CompanyName?
+		public let name: PersonaDataEntryName?
 
-		public let emailAddresses: OrderedSet<PersonaData.EmailAddress>?
-		public let phoneNumbers: OrderedSet<PersonaData.PhoneNumber>?
-		public let urls: OrderedSet<PersonaData.AssociatedURL>?
```

Comment:
None of these values (types) were ever shown to user, so safe to delete. Will be reintroduced later in Sargon, and might be reworked a little bit.

- `urls`
- `companyName`
- `creditCards`
- `postalAddresses`